holoviews/plotting/bokeh/callbacks.py

```diff
@@ -1191,13 +1191,24 @@ def _watch_position(self):
               }
               if (!xs || !ys) { return }
               for (const i of indices) {
-                const tx = xs[i]
+                let ix = xs[i]
+                let iy = ys[i]
+                if (typeof ix === 'number') {
+                  ix = [ix]
+                  iy = [iy]
+                } else {
+                  while (ix.length && (typeof ix[0] !== 'number')) {
```

Comment:
```suggestion
                  while (Array.isArray(ix) && (typeof ix[0] !== 'number')) {
```

`length` seems ambiguous

Reply:
No, the length check is intentional.

Reply:
If it's not an array or a number we have other problems I suppose.